apps/condo/pages/contact/[id]/index.tsx

```diff
@@ -272,6 +287,9 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
 
     await prefetchOrganizationEmployee({ client, context, userId: user.id })
 
+    const { id: contactId } = params as { id: string }
+    await prefetchContact({ client, contactId })
```

Comment:
maybe we should add a 403 page so we can redirect from ssr to this page? Then we can check access in ssr, what do you think?

(not in this pr)

Reply:
We should have some uniform behavior if the user has access to the data, but another organization is selected.
We should decide what behavior should be done everywhere

But probably not in this pr

Reply:
Yes, it's not a part of this pr